text/0000-add-cow-to-prelude.md

```diff
@@ -0,0 +1,138 @@
+- Feature Name: moooooooooooooooo
+- Start Date: 2019-04-01
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Re-export `std::borrow::Cow` from the standard library, along with each of its
+variants, to encourage more widespread use. To avoid backwards compatibility
+issues, these types/variants will need to be exported using more unique aliases.
+
+# Motivation/Explanation
+
+`std::borrow::Cow` is one of the most under-used portions of the standard
+library. Many issues that newcomers understanding ownership could be improved if
+`Cow` were used more prolifically throughout the ecosystem. The easiest way for
+us to encourage this is to remove the need to explicitly import `Cow` by
+exporting it from prelude.
+
+Anything exported in the prelude is generally expected to refer to what it
+refers to in the prelude. If a crate defines its own type called `Ok`, it will
+likely be confusing to readers, and will break any macros which don't refer to
+`std::result::Result::Ok` through its full path.
+
+Unfortunately, `Cow` is an extremely overloaded term in Rust today, so we can't
+reasonably export that identifier in the prelude. Applications today might be
+using this name to refer to:
+
+- California, Oregon, Washington (the most common usage)
+- Can of Worms
+- City of Waterloo
+- Crush of the Week
+- Container on Wheels
+- Cost of Waste
+
+to name a few. For this reason, the re-export of this type must be something
+which is unlikely or impossible to appear in Rust programs today. For this
+reason, the export will be defined as `pub use core::borrow::Cow as 🐄;`. As the
```

Comment:
Please provide the code point value for 🐄 just like 🐮, ☭, Ferris and Steve below